integration/spark-common/src/main/scala/org/apache/spark/util/CarbonReflectionUtils.scala

```diff
@@ -296,7 +296,7 @@ object CarbonReflectionUtils {
           classOf[LogicalPlan],
           classOf[Seq[Attribute]],
           classOf[SparkPlan])
-      method.invoke(dataSourceObj, mode, query, query.output, physicalPlan)
+      method.invoke(dataSourceObj, mode, query, query.output.map(_.name), physicalPlan)
```

Comment:
The parameters of 'writeAndRead' method had been changed, please see: [SPARK-PR#22346](https://github.com/apache/spark/pull/22346)